main/what-is-agoric.md

```diff
@@ -1,7 +1,33 @@
 # What is Agoric?
 
-Agoric is a layer 1, proof-of-stake public blockchain designed to enable developers to rapidly build safe smart contracts in the most popular programming language on the planet, JavaScript.
+Agoric is a Cosmos-based Layer 1 blockchain for building cross-chain
+smart contracts in JavaScript. The platform’s asynchronous,
+multi-block architecture uniquely enables developers to orchestrate
+liquidity and services across the growing multi-chain universe, making
+Agoric ideal for building software with seamless user experiences. The
+native token BLD is used for staking and governance. The
+over-collateralized IST stable token is used to pay network fees.
 
-The Agoric blockchain and crypto ecosystem is the smart contract platform that can quickly bring millions of developers to the DeFi frontier. Agoric’s hardened JavaScript makes blockchain programming accessible to the 10+ million JavaScript developers. Hardened JavaScript provides the safe, stable environment that developers need to build, deploy, and operate sophisticated Dapps, NFTs, and DeFi markets. The Agoric public blockchain, part of the Cosmos ecosystem, is built upon the battle-tested Tendermint Proof of Stake (PoS) consensus system with native IBC support. Moreover, the Agoric programming model, supported by our native market infrastructure, is designed to enable the formation of a cryptoeconomic standard library with the same exponential composability that led to the explosive growth of the Node.js, React.js, and other JavaScript ecosystems.
+The Agoric blockchain is built on the CometBFT consensus system with
+native Inter-Blockchain Communication Protocol (IBC) support. Smart
+contracts can seamlessly integrate 3rd party libraries, so a contract
+could invoke an operation on Avalanche via Axelar’s GMP the same way
+it does on Osmosis via IBC. Agoric smart contracts are long-lived
```

Comment:
Do we want to include this even though the docs don't include any way for contracts to use IBC?

Reply:
@sufyaankhan said yes; we've been public that it's coming